python/ray/tests/test_state_api_summary.py

```diff
@@ -382,13 +382,14 @@ def verify():
         summary = summary["cluster"]["summary"]
         actor_summary = None
         infeasible_summary = None
+        print(summary)
         for actor_class_name, s in summary.items():
             if ".Actor" in actor_class_name:
                 actor_summary = s
             elif ".Infeasible" in actor_class_name:
                 infeasible_summary = s
 
-        assert actor_summary["state_counts"]["PENDING_CREATION"] == 1
+        assert actor_summary["state_counts"]["DEPENDENCY_UNREADY"] == 1
```

Comment:
@rkooo567 I fail to figure out why the change in this PR has an impact on this task - do you have any ideas? 
Is the change expected if we are now getting actors from dead jobs by default? Might be some data mismatch from gcs data.

Reply:
Hmm this doesn't make sense at all? We don't have any actor that even has an dependency... we may need to dig deeper